pkg/kubectl/resource_printer.go

```diff
@@ -599,6 +615,11 @@ func printPod(pod *api.Pod, w io.Writer, options PrintOptions) error {
 func printPodBase(pod *api.Pod, w io.Writer, options PrintOptions) error {
 	name := pod.Name
 	namespace := pod.Namespace
+	kind := options.KindName
```

Comment:
I try to set `KindName` [in these lines](https://github.com/juanvallejo/kubernetes/blob/e10a59f93504c87367bf2f052af3639f73aa8b63/pkg/kubectl/cmd/get.go#L339-346) as an alias. If an alias for the specific resource is not found, `KindName` is simply set as the long resource name